src/customize-pop/application-settings.md

```diff
@@ -0,0 +1,63 @@
+# Application Settings
+
+- [`Applications` settings](/customize-pop/application-settings.md#applications): Manage application permissions, allow system integration features, specify default handlers, and view disk space usage per app. These settings may not be available for all applications, depending on the functionality or access required by the application.
+- [`Notifications` settings](/customize-pop/application-settings.md#default-applications): Control which applications display notifications, what notifications are displayed, and where they are displayed.
+- [`Default Applications` settomgs](/customize-pop/application-settings.md#default-applications) Specify applications that should always open certain file types.
+
+---
+
+## Applications
+
+Options to control application access to system features and resources are located in the `Settings` ➞ `Applications` tab.
+
+![Application Settings](/images/application-settings/application-settings.png)
+
+| Option | Function |
+|---------|----------|
+| Permissions & Access | View the data and system services requested or required by an application. Permissions can include access to system devices, using the system's network connection, reading and writing to the file system, and the ability to change settings. Many applications include built-in permissions that cannot be modified by the user. View permission granted to an application by clicking the `Built-in Permissions` button. |
+| Integration | Disable or enable system-wide features used by an application. These features include the ability to appear in search, display notification, and running in the background. |
```

Comment:
> display notification

Should be either `display notifications` or `display a notification`.

> and running in the background.

Might want to make this `and run in the background`.